package.json

```diff
@@ -6,18 +6,16 @@
     "node": "8.12.0"
   },
   "scripts": {
-    "build": "yarn run build-client && yarn run build-server && yarn run copy-reactapp",
+    "build": "lerna clean -y && lerna bootstrap && lerna run build && yarn run copy-reactapp",
     "copy-reactapp": "node copy-reactapp.js",
-    "build-client": "yarn --cwd client install && yarn --cwd client run build",
-    "build-server": "yarn --cwd server install && yarn --cwd server run build",
```

Comment:
we don't need separate build scripts in the root `package.json` anymore, sicne `lerna run build` builds all packages automatically